concepticondata/conceptlists/RonaTas-2011-431.tsv

```diff
@@ -0,0 +1,432 @@
+ID	NUMBER	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS
+RonaTas-2011-431-1	1	carpenter	1042	CARPENTER
+RonaTas-2011-431-2	2	cannon, catapult	3202	CANNON
+RonaTas-2011-431-3	3	to fondle, pet, caress	3780	PET
+RonaTas-2011-431-4	4	stream, current	2935	CURRENT (STREAM)
+RonaTas-2011-431-5	5	false, imitation	3069	FALSE
+RonaTas-2011-431-6	6	pied (of an animals coat)		
+RonaTas-2011-431-7	7	apple	1320	APPLE
+RonaTas-2011-431-8	8	barrel (wooden container andstandard quantity)	2548	BARREL
+RonaTas-2011-431-9	9	to nurse, take care of	2472	SUCKLE (BREASTFEED, NURSE)
```

Comment:
I would possibly also unmap this. The meaning in the original list seems to refer more towards an educational nursing (something like german "großziehen"), while `SUCKLE` is quite specific for breastfeeding.

Reply:
Resolved in 029e10648eb0e8351eec8b62d2130cb9250afc10